app/containers/cabalSettings.js

```diff
@@ -71,6 +71,15 @@ class CabalSettingsContainer extends React.Component {
                   <input type='text' value={`cabal://${this.props.cabal.addr}`} />
                 </div>
               </div>
+              <div className='cabal-settings__item'>
```

Comment:
I think it would be great if this was the first option in the list... above the notification setting... 

also, want to give the field a placeholder so it's clear what do with the field? something like "Set local alias..." or "My Favorite Cabal" ? :D